packages/components/psammead-useful-links/src/index.stories.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import {
+  UsefulLink,
+  UsefulLinksLi,
+  UsefulLinksUl,
+  UsefulLinkItem,
+} from './index';
+import SectionLabel from '../../psammead-section-label/src';
+import notes from '../README.md';
+
+const usefulCaptions = [
```

Comment:
This could take the structure of 
```jsx
const usefulCaptions = [{ name: 'Some Caption', url: 'http://somelink'}]
```
Which becomes much easier to work with IMO

Reply:
Thank you, done now :)